Modules/_ssl.c

```diff
@@ -4427,6 +4427,12 @@ _ssl__SSLContext_load_dh_params_impl(PySSLContext *self, PyObject *filepath)
     FILE *f;
     DH *dh;
 
+#if defined(MS_WINDOWS) && defined(_DEBUG)
+    PyErr_SetString(PyExc_RuntimeError,
+                    "unable to load_dh_params on Windows debug build");
```

Comment:
For the messages, maybe something like `"FUNC_NAME: unavailable on Windows debug build"` or "cannot use FUNC_NAME on Windows debug build"?

Reply:
Nobody should see them, so I'm not hugely concerned, but if there's a reason to modify and run CI again then I'll do them.

Reply:
Not really! just that I found it funny to have the funcname acting as the "verb" in the sentence :')